plenum/common/messages/fields.py

```diff
@@ -368,16 +368,13 @@ def __init__(self, *args, **kwargs):
 
 
 class TimestampField(FieldBase):
-    _base_types = (float, int)
+    _base_types = (int,)
+    _oldest_time = 1499906902
```

Comment:
What network do you mean? Alpha?
BTW please be careful with adding new config variables if it's not intended to be changed by users (please see one of the recent emails from Daniel)